otwcode/otwarchive
# AO3-4810 Extend tests for PromptsController

## Issue

https://otwarchive.atlassian.net/browse/AO3-4810

## Purpose

prompts_controller currently has 65.17% test coverage on Coveralls:
https://coveralls.io/builds/9474724/source?filename=app%2Fcontrollers%2Fprompts_controller.rb
We'd like to increase that, with an ultimate goal of 90% or greater.
How to test:
Use Coveralls to confirm that the coverage percentage has increased:
1. Go to https://coveralls.io/github/otwcode/otwarchive?branch=master 
2. In the "Search" field near the "Source Files on Master" heading, enter the name of the controller: prompts_controller
